.github/workflows/redhat-csp-download.yaml

```diff
@@ -0,0 +1,35 @@
+name: Test redhat-csp-download
+
+on:
+  push:
+    paths:
+      - .github/workflows/redhat-csp-download.yaml
+      - redhat-csp-download/**
+  pull_request:
```

Comment:
@garethahealy So one other thing to keep in mind here is that in this repository, the `pull_request` checks will always fail because this action needs access to secrets. Since `pull_requests` only have read access to repositories from forks.. they won't actually have access to the secrets.

There are workarounds for this that I'm going to try to get an example of out for, but for now it might be best to just remove the `pull_request` section from this workflow.

Reply:
@tylerauerbeck ; the action skips if the username/password is not provided. I'd prefer to keep this in for that reason, as at least on PRs, it'll build the image and give any early warnings and on the master, it'll run the playbook.